.github/workflows/framework-tests.yaml

```diff
@@ -75,10 +75,10 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Set up Go 1.20
+      - name: Set up Go
```

Comment:
Seems to me that there's no benefit in repeating the version here and it's just an easy place for it to get out of sync with the actual version just below.